### PR TITLE
fix(create-app): fix user-settings routing in create-app

### PR DESCRIPTION
**Pull request by kiranpatel11**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

File: packages/create-app/templates/default-app/packages/backend/src/plugins/scaffolder.ts [hunk 1]
```diff
@@ -1,21 +1,13 @@
 import {
   CookieCutter,
   createRouter,
-  FilePreparer,
```
Comment (freben) [inline packages/create-app/templates/default-app/packages/backend/src/plugins/scaffolder.ts:4]: these changes are unrelated

File: packages/create-app/templates/default-app/packages/app/src/App.tsx [hunk 2]
```diff
@@ -15,6 +15,7 @@ import { Router as DocsRouter } from '@backstage/plugin-techdocs';
 import { Router as ImportComponentRouter } from '@backstage/plugin-catalog-import';
 import { Router as TechRadarRouter } from '@backstage/plugin-tech-radar';
 import { SearchPage as SearchRouter } from '@backstage/plugin-search';
+import { Router as SettingsRouter } from '@backstage/plugin-user-settings';
```
Comment (freben) [inline packages/create-app/templates/default-app/packages/app/src/App.tsx:18]: changes in create-app template should have a changeset, with some short instructions for people how to apply the corresponding change manually inside their own repo

File: packages/create-app/templates/default-app/packages/backend/src/plugins/scaffolder.ts (hunk 1 above)
Comment (kiranpatel11) [reply packages/create-app/templates/default-app/packages/backend/src/plugins/scaffolder.ts]: reverted unrelated changes from this PR

File: packages/create-app/templates/default-app/packages/app/src/App.tsx (hunk 2 above)
Comment (kiranpatel11) [reply packages/create-app/templates/default-app/packages/app/src/App.tsx:18]: included changeset

File: .changeset/ninety-turtles-fix.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+fix routing and config for user-settings plugin
```
Comment (freben) [inline .changeset/ninety-turtles-fix.md:5]: OK two final asks.

First: You need to rebase on the most recent master; we mistakenly caused a build error in master yesterday and it's fixed now.

Second: When doing app template changes, it's nice to add a few lines here (that end up in the changelog) that inform users how to make the corresponding change in their create-app version of the app. Something like

To make the corresponding change in your local app, add the following in your App.tsx:

```ts
import { Router as SettingsRouter } from '@backstage/plugin-user-settings';
...
<Route path="/settings" element={<SettingsRouter />} />
```

and the following to your plugins.ts:

```ts
export { plugin as UserSettings } from '@backstage/plugin-user-settings';
```

Comment (kiranpatel11) [reply .changeset/ninety-turtles-fix.md]: @freben its done.

**Review by Rugvip — APPROVED** [review]
Thanks you! :grin:
